### PR TITLE
for codesy/chrome-extension#68 - tweak jpm usage

### PR DESCRIPTION
**Pull request by groovecoder**
This helped me get the firefox extension working more easily with jpm.

### CODE REVIEW

File: src/firefox/index.coffee [hunk 1]
```diff
@@ -56,7 +56,7 @@ pageMod.PageMod {
 # codesy home page
 pageMod.PageMod {
   include : [
-    /.*localhost.*/
+    /.*127.0.0.1:8443.*/
```
Comment (jdungan) [inline src/firefox/index.coffee]: This file and the firefox package.json are meant to stay production ready.

You can change the dev_domain variable in the gulp file to achieve this.

**Comment by groovecoder** [comment]
Updated. I removed the lingering `localhost` line in `src/firefox/index.coffee`.

File: README.rst [hunk 2]
```diff
@@ -30,7 +28,7 @@ To use the Firefox Addon
 A  new directory called firefox will be created. In a new terminal window:
 
 2. cd firefox     
-3. run 'jpm run --profile your-profile'  (this runs a new instance of firefox with your default profile so your bookmarks, etc. are available)
+3. run './node_modules/.bin jpm run --profile your-profile'  (this runs a new instance of firefox with your default profile so your bookmarks, etc. are available)
```
Comment (jdungan) [inline README.rst]: This doesn't work for me.

File: src/firefox/index.coffee [hunk 3]
```diff
@@ -56,7 +56,6 @@ pageMod.PageMod {
 # codesy home page
 pageMod.PageMod {
   include : [
-    /.*localhost.*/
```
Comment (jdungan) [inline src/firefox/index.coffee:59]: Oops.  This one has to stay.  I haven't created a gulp task to morph javascript (yet).

File: README.rst (hunk 2 above)
Comment (groovecoder) [inline README.rst]: Did you do `npm install` again? I moved `jpm` into the `package.json` so it should install `jpm` when you do `npm install`

File: src/firefox/index.coffee (hunk 3 above)
Comment (groovecoder) [inline src/firefox/index.coffee:59]: Restored.

**Pull request merged**